tools/hicup/.shed.yml

```diff
@@ -0,0 +1,17 @@
+name: hicup
+owner: joachimwolff
+description: A tool for mapping and performing quality control on Hi-C data.
+
+long_description: |
+  HiCUP is a tool to process HiC data developed by the Babraham Bioinformatics Institute, Cambridge, UK.
+
+categories:
+  - HiC 
```

Comment:
Just as information, these categories needs to match the ones in the TS.
A HiC category is a good idea, but we need to propose this to the devs.